### ISSUE TITLE
My config uses some nixos and I was wondering how to submit it

### ISSUE THREAD

**Issue by Brettm12345** [zdharma/zinit-configs#24]
My config is broken up between a few files.

[`shell.nix`](https://github.com/Brettm12345/nixos-config/blob/master/modules/applications/shell.nix) which contains my global shell configuration between `bash`, `fish` and `zsh`.

as well as everything in this directory
https://github.com/Brettm12345/nixos-config/tree/master/modules/applications/zsh

**Comment by Brettm12345** [comment]
I just checked and I saw you were using a dockerfile. Maybe we could add support for nixos with docker

**Comment by Brettm12345** [comment]
Update all of my shell configuration has been moved to [here](https://github.com/Brettm12345/nixos-config/tree/master/modules/applications/shells)

**Comment by psprint** [comment]
Yes, the docker file would be have to point to nixos. I think that's totally doable, I'll try to approach this.
